docs/guides/quick-start/miranum-connect/miranum-connect-quickstart.md

```diff
@@ -0,0 +1,31 @@
+---
+id: miranum-connect-quickstart
+title: Miranum Connect Quickstart
+sidebar_label: Overview
+description: "This section introduces the concept of Miranum Connect and how to use it."
+---
+
+# Miranum Connect Quickstart Guide
```

Comment:
⛏️ I noticed that the GitHub Templates are not included in the guides yet. I think we should add them so more people can make use of them. Wdyt?